exercises/tutorial_halfday/ex8_tiled-matrix-transpose.cpp

```diff
@@ -231,31 +241,30 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
 
   //
   // This policy loops over tiles sequentially while collapsing inner loops
-  // into a single OpenMP parallel for loop enabling parallel loads/reads
+  // into a single OpenMP loop enabling parallel loads/reads
   // to/from the tile.
   //
-#if defined(ENABLE_POLICY)
   using KERNEL_EXEC_POL_OMP2 =
     RAJA::KernelPolicy<
-      RAJA::statement::Tile<1, RAJA::statement::tile_fixed<TILE_SZ>, RAJA::seq_exec,
-        RAJA::statement::Tile<0, RAJA::statement::tile_fixed<TILE_SZ>, RAJA::seq_exec,
+      RAJA::statement::Tile<1, RAJA::statement::tile_fixed<TILE_SZ>, 
+                               RAJA::seq_exec,
+        RAJA::statement::Tile<0, RAJA::statement::tile_fixed<TILE_SZ>, 
+                                 RAJA::seq_exec,
           RAJA::statement::Collapse<RAJA::omp_parallel_collapse_exec,
                                     RAJA::ArgList<0, 1>,
             RAJA::statement::Lambda<0>
           > //closes collapse
         > // closes Tile 0
       > // closes Tile 1
     >; // closes policy list
-#endif
 
-  ///
-  /// TODO...
-  ///
-  /// EXERCISE:
-  ///
-  ///   Implement the matrix tranpose kernel using the RAJA kernel API
-  ///   and the kernel policy above.
-  ///
+  RAJA::kernel<KERNEL_EXEC_POL_OMP2>(
```

Comment:
I think the solution slipped in here, this was supposed to be TODO right?

Reply:
Actually, that was intentional. I think the omp collapse version is good for folks to study, but I was trying to trim down the number of exercises to work on so I removed it as a "TO DO". Also, I was trying to make it so the "exercise" and "solution" file for each exercise are only different in the parts that need to be filled in as part of the exercise -- easy to use diff, etc.